sunpycube/cube/tests/test_cube.py

```diff
@@ -107,137 +98,154 @@ def test_choose_x_slice():
     ios = cube._choose_x_slice(11)  # integer, over range slice
 
     qus = cube._choose_x_slice(-1 * u.deg)  # quantity, under
-    qis = cube._choose_x_slice(1.5 * u.deg)  # quantity, in
+    qis = cube._choose_x_slice(0.5 * u.deg)  # quantity, in
     qos = cube._choose_x_slice(8 * u.deg)  # quantity, over
 
     f = cube._choose_x_slice(0.4)  # no units given
 
     assert ius is None
-    assert np.all(iis == [[2, 4, -1], [4, 5, 3]])
+    assert np.all(iis == [[2, 4, 5, 1], [10, 5, 2, 2], [10, 3, 3, 0]])
     assert ios is None
 
     assert qus is None
-    assert np.all(qis == [[4, 3, 3], [1, 2, 0]])
+    assert np.all(qis == [[1, 2, 3, 4], [2, 4, 5, 3], [0, -1, 2, 3]])
     assert qos is None
 
     assert f is None
 
 
+def test_slice_to_map_with_time():
+    with pytest.raises(cu.CubeError):
+        cube.slice_to_map(0)
+    with pytest.raises(cu.CubeError):
+        cube.slice_to_map((0, 3))
+
+
+def test_slice_to_map_no_time():
+    m0 = cubem.slice_to_map(0)
+    m1 = cubem.slice_to_map((0, 3))
+    assert np.all(m0.data == cubem.data[0])
+    assert np.all(m1.data == cubem.data.sum(0))
+
+
 def test_wavelength_axis():
     cube_wavelength_axis = cube.wavelength_axis()
     cubem_wavelength_axis = cubem.wavelength_axis()
     f1, u1 = cube_wavelength_axis.value, cube_wavelength_axis.unit
     f2, u2 = cubem_wavelength_axis.value, cubem_wavelength_axis.unit
     # the e-11 are the conversions from angstrom to meters
-    assert np.allclose(f1, [0, 2.0e-11, 4.0e-11])
-    assert np.allclose(f2, [0, 2.0e-11, 4.0e-11])
+    assert np.allclose(f1, [0.00000000e+00,  3.00000000e-11,  6.00000000e-11])
```

Comment:
@Cadair is asset_allclose in pytest or did you mean numpy.testing.assert_allclose?